instrumentation_test.go

```diff
@@ -226,10 +226,10 @@ func TestWithSampler(t *testing.T) {
 		sc, err := convertSamplerToConfig(c.sampler)
 		assert.NoError(t, err)
 		assert.Equal(t, sc.Samplers, sampling.DefaultConfig().Samplers)
-		assert.Equal(t, sc.ActiveSampler, sampling.ParentBasedID)
+		assert.Equal(t, sampling.ParentBasedID, sc.ActiveSampler)
```

Comment:
Shouldn't the one line above also be fixed even though the linter has not catched it?